.github/workflows/update-wpt.yml

```diff
@@ -0,0 +1,67 @@
+name: WPT update
+
+on:
+  schedule:
+    # Run once a week at 12:00 AM UTC on Sunday.
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  wpt-subsystem-update:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        subsystem:
+          - url
+
+    steps:
+      - uses: actions/checkout@v4@<hash>
```

Comment:
Can you add the hash?

Reply:
Do you have any idea how I can get this hash?

Reply:
check the latest v4 tag, you get the commit hash, and add a comment with the tag

Reply:
Or check any of the other workflows in use

Reply:
i added, thanks